### ISSUE TITLE
Feature request: assign multiple files to a category

### ISSUE THREAD

**Issue by supersunken** [projectsend/projectsend#353]
Would like to see the ability to assign multiple files that have been uploaded to a new category. Currently have to click through each file to assign it to a category.

**Comment by arachnetech** [comment]
I agree - if you don't assign categories while uploading files then it takes a long time to go back and edit them all.

Similarly, it would be great to be able to select multiple files and then assign them to clients or client groups.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
